src/sentry/static/sentry/app/utils/stream.tsx

```diff
@@ -20,7 +20,7 @@ type QueryObj = {
 export function queryToObj(queryStr: string): QueryObj {
   const text: string[] = [];
 
-  const queryItems = queryStr.match(/\S+:"[^"]*"?|\S+/g);
+  const queryItems = (queryStr || '').match(/\S+:"[^"]*"?|\S+/g);
```

Comment:
What about setting it as a default param?

Reply:
That's probably better